ksqldb-engine/src/main/java/io/confluent/ksql/physical/pull/operators/WhereInfo.java

```diff
@@ -433,25 +429,6 @@ private static KeyAndWindowBounds extractWhereClauseTarget(
     return Range.range(lower, lowerType, upper, upperType);
   }
 
-  private static Object coerceKey(
```

Comment:
Why don't we need this method anymore?

Reply:
it was moved into `resolveKey` logic, which is more generic. This method wasn't actually used anywhere